src/Algebra/Properties/Monoid/Sum.agda

```diff
@@ -70,3 +71,21 @@ sum-replicate-idem idem n = trans (sum-replicate n) (×-idem idem n)
 sum-replicate-zero : ∀ n → sum {n} (replicate 0#) ≈ 0#
 sum-replicate-zero zero    = refl
 sum-replicate-zero (suc n) = sum-replicate-idem (+-identityˡ 0#) (suc n)
+
+------------------------------------------------------------------------
+-- When summing over a `Vector`, we can pull out last element
```

Comment:
I think using a section heading for this overkill. I would just attach the comment to the proof. i.e. delete the empty line and remove the long line of `------`s

Reply:
Again, the original had this as one of a number of lemmas about `sum`, each with its own section heading, and I had just carried that one over to its new home. Will clean up.

Reply:
Done, except that I left the blank line in, to lay off the lemma from the preceding ones. Looked better!